### PR TITLE
Skip flaky test on safari

### PR DESCRIPTION
**Pull request by zhouyx**
#11382, #11459

### CODE REVIEW

File: test/integration/test-user-error-reporting.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
     depositRequestUrl,
 } from '../../testing/test-helper';
 
-describe.configure().run('user-error', function() {
+describe.configure().skipSafari().run('user-error', function() {
```
Comment (rsimha-amp) [inline test/integration/test-user-error-reporting.js]: Perhaps put in a `TODO` here with the original issue #?

**Comment by zhouyx** [comment]
The integration tests are so flaky : ( I can't get a green build to merge this

**Comment by rsimha-amp** [comment]
Hmm. https://status.saucelabs.com doesn't indicate anything. The failures appear to be in `amp-bind` integration tests. cc @choumx

**Comment by rsimha-amp** [comment]
Since this PR is just skipping a flaky test, maybe @erwinmombay can force-merge it for you?

**Comment by zhouyx** [comment]
Yes! green build

**Comment by rsimha-amp** [comment]
Yay! Thanks for making sure this went through.

**Pull request merged**